From 1ae04305b649a16fabfe63e52a9d9e393ff9ea59 Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Mon, 20 Mar 2017 14:23:19 +0800
Subject: [PATCH] arm64: dts: rk3368-p9&sheep: disable uart2

Disable uart2 since gpio mux on uart with sdmmc, and rk3368 use
fiq debugger, the uart2 could set to be disabled.

Change-Id: I2d784ccd6cf7526afc0f3bae54914e05febf91a6
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3368-p9.dts    | 4 ----
 arch/arm64/boot/dts/rockchip/rk3368-sheep.dts | 4 ----
 2 files changed, 8 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3368-p9.dts b/arch/arm64/boot/dts/rockchip/rk3368-p9.dts
index 8823032a365c..3cbc43599e4d 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368-p9.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3368-p9.dts
@@ -624,10 +624,6 @@
 	status = "okay";
 };
 
-&uart2 {
-	status = "okay";
-};
-
 &saradc {
 	status = "okay";
 };
diff --git a/arch/arm64/boot/dts/rockchip/rk3368-sheep.dts b/arch/arm64/boot/dts/rockchip/rk3368-sheep.dts
index c7834da95809..257ee5e46a96 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368-sheep.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3368-sheep.dts
@@ -512,10 +512,6 @@
 	status = "okay";
 };
 
-&uart2 {
-	status = "okay";
-};
-
 &u2phy {
 	status = "okay";
 
-- 
2.35.3

